.github/workflows/ci.yaml

```diff
@@ -47,6 +47,9 @@ jobs:
       - name: Integration Tests
         run: bazel --batch test //test/integration/...
 
+      - name: Showcase Tests
```

Comment:
is this only generating the showcase client libraries at this moment?

Reply:
Correct - and only once merged. More will be needed, but the intent was to get it executed by CI asap.